### ISSUE TITLE
extra_msg mismatch?

### ISSUE THREAD

**Issue by Bluscream** [TeaSpeak/TeaSpeak#20]
Normal Server:
```csharp
[OUT] banlist return_code=1:1
[ IN] error id=1281 msg=database\sempty\sresult\sset return_code=1:1
```

TeaSpeak Server:
```csharp
[OUT] banlist return_code=1:n
[ IN] error id=1281 msg=sql\sempty\sresult\sset return_code=1:n extra_msg=empty extramsg=empty
```

**Comment by WolverinDEV** [comment]
hmm yeah i think i should send extra_msg once :D

**Comment by Bluscream** [comment]
I don't think that extra_msg is needed at all hence the official server doesn't use it and the client doesn't show it.

**Comment by WolverinDEV** [comment]
Fixed in 1.0.11-Beta (removed the `extramsg` pair)
